browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -302,6 +344,36 @@ void SpeedreaderTabHelper::DidStopLoading() {
   }
 }
 
+void SpeedreaderTabHelper::DOMContentLoaded(
+    content::RenderFrameHost* render_frame_host) {
+  if (!PageWantsDistill(distill_state_))
+    return;
+
+  constexpr int kIsolatedWorldId = content::ISOLATED_WORLD_ID_CONTENT_END + 1;
```

Comment:
just making sure i understand: this will run in the same isolated world as Translate? https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/chrome_isolated_world_ids.h;l=13?q=ISOLATED_WORLD_ID_CONTENT_END&ss=chromium

Reply:
Yep, but it i can change it if needed.

Reply:
probably best to use a new world